flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -112,7 +113,12 @@ class DataStreamOverAggregate(
               "condition.")
         }
       case _: RowTimeType =>
-        throw new TableException("OVER Window of the EventTime type is not currently supported.")
+        if (overWindow.lowerBound.isUnbounded && overWindow.upperBound.isCurrentRow) {
+          createUnboundedAndCurrentRowEventTimeOverWindow(inputDS)
```

Comment:
I see, so this PR addresses the `ROW` case.
I'll push out a hotfix. 

Thanks for the notification @sunjincheng121!